resources/views/layouts/default/pireps/show.blade.php

```diff
@@ -15,7 +15,7 @@
            class="btn btn-outline-info">View SimBrief</a>
       @endif
 
-      @if(!$pirep->read_only)
+      @if(!$pirep->read_only && $pirep->user_id === Auth::id())
```

Comment:
One thing - in the controller for the view, set a `user` and then here use `$pirep->user_id === $user->id`. I'd rather the $user be set in the controllers.